cmd/kubeseal/main.go

```diff
@@ -22,6 +22,10 @@ import (
 
 	// Register v1.Secret type
 	_ "k8s.io/client-go/pkg/api/install"
+
+	// Register Auth providers
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
```

Comment:
I didn't even notice that. I'll change to use that. Thanks for the quick feedback!